src/ComposerRunner/SymfonyProcessComposerRunner.php

```diff
@@ -15,9 +15,7 @@ public function licenses(string $projectPath): string
         $process = Process::fromShellCommandline('SHELL_VERBOSITY=0 composer licenses --format=json', $projectPath);
         $process->run();
         if (!$process->isSuccessful()) {
-            throw FailedRunningComposer::withReason(
-                \sprintf('Failed to run "%s" (%d)', \addcslashes($process->getCommandLine(), '"'), $process->getExitCode()),
-            );
+            throw FailedRunningComposer::withCommand(\addcslashes($process->getCommandLine(), '"'));
```

Comment:
```suggestion
            throw FailedRunningComposer::withCommand($process->getCommandLine());
```

Don't need to escape as we're now just appending it to a prefix rather than wrapping it in quotes.